src/Tables/trainer_tables.c

```diff
@@ -1004,6 +1004,106 @@ const struct TrainerMonNoItemDefaultMoves sParty_Route8_CoolTrainerHarriet[] = {
 };
 // #endregion
 
+// #region Forgotten Manse
+const struct TrainerMonNoItemDefaultMoves sParty_ForgottenManse_ChannelerMona[] = {
+    { .lvl = 25, .species = SPECIES_HATENNA },
+    { .lvl = 25, .species = SPECIES_DUSKULL },
+    { .lvl = 25, .species = SPECIES_MISDREAVUS }
+};
+
+const struct TrainerMonNoItemDefaultMoves sParty_ForgottenManse_BurglarChad[] = {
+    { .lvl = 27, .species = SPECIES_SNEASEL },
+    { .lvl = 27, .species = SPECIES_BOLTUND },
```

Comment:
Here you end the array with , and don't in the others ones

Reply:
oops, fixed for consistency (both)